docs/references/javascript/clerk/user-button.mdx

```diff
@@ -87,7 +87,7 @@ function mountUserButton(node: HTMLDivElement, props?: UserButtonProps): void;
   cells: [
   "props?",
   <a href="#user-button-props"><code>UserButtonProps</code></a>,
-  "The properties to pass to the `<UserButton />` component",
+  "The properties to pass to the {"<UserButton />"} component",
```

Comment:
@alexisintech As this one is just a string, no need for the fix here

Reply:
i'm gonna rip my hair out